File: gymcrm.web/src/modules/billing/types/subscription.ts
// Mirrors GymCRM.BillingAPI.Models.DTOs.Subscription 1:1.
export interface Subscription {
    id: string;
    memberAccountGuid: string;
    planType: number;
    status: number;
    startDate: string;
    nextRenewalDate?: string;
    dateCreated: string;
    dateModified: string;
}

File: GymCRM.BillingAPI/Models/DTOs/Subscription.cs
namespace GymCRM.BillingAPI.Models.DTOs;

public class Subscription
{
    public Guid Id { get; set; }
    public Guid MemberAccountGuid { get; set; }
    public int PlanType { get; set; }
    public int Status { get; set; }
    public DateTime StartDate { get; set; }
    public DateTime? NextRenewalDate { get; set; }
    public DateTime DateCreated { get; set; }
    public DateTime DateModified { get; set; }
}
